prover/docs/03_launch.md

```diff
@@ -0,0 +1,92 @@
+# Running provers
+
+## Preparing
+
+First, run the following command:
+
+```
+zk env prover-local
+```
+
+It will create a config similar to `dev`, but with:
+
+- Proof sending mode set to `OnlyRealProofs`
+- Prover mode set to `Local` instead of `GCS`.
+
+You can always switch back to dev config via `zk env dev`.
+
+## Key generation
+
+This operation should only be done once; if you already generated keys, you can skip it.
+
+The following command will generate the required keys:
+
+```
+zk f cargo run --features gpu --release --bin key_generator -- generate-sk-gpu all --recompute-if-missing
+```
+
+With that, you should be ready to run the prover.
+
+## Running
+
+Important! Generating a proof takes a lot of time, so if you just want to see whether you can generate a proof, do it
+against clean sequencer state (e.g. right after `zk init`).
+
+We will be running a bunch of binaries, it's recommended to run each in a separate terminal.
```

Comment:
FYI -- there is also "zk status prover" command, that will show you the "progress" of your proof (how many circtuts are done etc).